CWE/decisions/change_primary_culture.txt

```diff
@@ -5,24 +5,107 @@ political_decisions = {
 		
 		potential = { 
 			NOT = { tag = ARG }
-			NOT = { tag = CHI }
-			capital_scope = { OR = { continent = north_america continent = south_america }  NOT = { pop_majority_culture = white_spanish } } 
-			primary_culture = white_spanish
+			NOT = { tag = CHL }
+			capital_scope = { 
+				OR = { 
+					continent = north_america 
+					continent = south_america 
+				}
+				NOT = {
+					OR = {
+						pop_majority_culture = mesoamerican_white_spanish
+						pop_majority_culture = andean_white_spanish
+						pop_majority_culture = platinean_white_spanish
+						pop_majority_culture = caribbean_white_spanish
+						AND = {
+							tag = BRZ
+							pop_majority_culture = white_portuguese
+						}
+					}
+				} 
+			} 
+			OR = {
+				primary_culture = mesoamerican_white_spanish
+				primary_culture = andean_white_spanish
+				primary_culture = platinean_white_spanish
+				primary_culture = caribbean_white_spanish
+				tag = BRZ
+			}
 			NOT = { has_country_flag = pink_tide }
 		} 
 		
 		allow = { 
-			government = democracy
-			OR = { ruling_party_ideology = progressive ruling_party_ideology = socialist } 
+			OR = { government = democracy government = democracy1 government = hms_government government = hms_government1 government = socialist_democracy }
+			OR = { ruling_party_ideology = progressive ruling_party_ideology = socialist ruling_party_ideology = communist ruling_party_ideology = communist_social } 
 			minorities_reform = minorities_affirmative_action
-			wars_of_national_liberation = 1
-		} 
+			invention = indigenismo
+		}
 		
 		effect = { 
 			prestige = 5
-			random_country = { limit = { tag = THIS THIS = { tag = BRZ }  } primary_culture = pardo_portuguese add_accepted_culture = white_portuguese }  
-			random_country = { limit = { NOT = { tag = BRZ } tag = THIS } primary_culture = mestizo add_accepted_culture = white_spanish }  
-			any_pop = { limit = { OR = { culture =  pardo_portuguese culture =  mestizo } } militancy = -2.0 consciousness = 1.0 } 
+			
+			random_country = {
+				limit = {
+					tag = THIS
+					THIS = { tag = BRZ }
+				}
```

Comment:
random_country = { limit = { tag = THIS } } does not work as the scope can only check for tags that exist other than THIS

Reply:
That's what it was before, I just made it more readable by parsing it out in a more 'clean' way. I'm not sure what would be a good alternative.

Reply:
Yeah the decision was broken previously. NWO used this a lot although it didn’t work.

This should do what it’s supposed to do:

random_owned = { limit = { owner = { tag = BRZ } } owner = { primary_culture … } }

Reply:
Alright, thanks.

Reply:
For some reason its not saying I updated it, but I definitely did.